src/Tools/Generator.php

```diff
@@ -76,6 +76,36 @@ public function processRoute(Route $route, array $rulesToApply = [])
         return $parsedRoute;
     }
 
+    protected function getBodyParameters(ReflectionMethod $method, array $tags)
+    {
+        /** @var ReflectionClass $cls */
+        $cls = collect($method->getParameters())
```

Comment:
I spent the better part of a minute trying to figure out what "cls" represents. Please rename the variable to something more meaningful that reflects the context. `$formRequestClass` works, I think.

Reply:
makes sense